boards/iotlab-m3/Makefile.dep

```diff
@@ -1,5 +1,12 @@
 ifneq (,$(filter netdev_default gnrc_netdev_default,$(USEMODULE)))
   USEMODULE += at86rf231
+  ifeq ($(filter gnrc_802154_basic_mac,$(USEMODULE)),)
+  USEMODULE += gnrc_nomac
```

Comment:
Is this PR still based on `gnrc_netdev`? Can you please port to `netdev2`? `gnrc_netdev` was deprecated in #5352.